jenkins/version-def.sh

```diff
@@ -17,6 +17,15 @@
 
 set -e
 
+PHASE_TYPE=regular
```

Comment:
would be nice to document what phases there are

Reply:
added comments for PHASE_TYPE